java/src/processing/mode/java/JavaEditor.java

```diff
@@ -327,6 +327,18 @@ public JMenu buildHelpMenu() {
 
     menu.addSeparator();
 
+    // Report a bug link opener
+    item = new JMenuItem(Language.text("menu.help.report"));
+    item.addActionListener(e -> Platform.openURL(Language.text("menu.help.report.url")));
+    menu.add(item);
+
+    // Ask on the Forum link opener
+    item = new JMenuItem(Language.text("menu.help.ask"));
+    item.addActionListener(e -> Platform.openURL(Language.text("menu.help.getting_started.url")));
```

Comment:
Hey @lassevonpfeil. I just caught while testing: the URL here should have been `menu.help.ask.url` instead of `menu.help.getting_started.url`. Would you mind making another PR for this small fix? Thanks! cc @catilac